frontend/src/pages/ErrorFetchingResources/ErrorFetchingResources.js

```diff
@@ -7,7 +7,8 @@ export default function ErrorFetchingResources() {
   const navigate = useNavigate();
 
   function navigateToHomepage() {
-    navigate("/");
+    // navigate("/");
```

Comment:
please rather remove the unused code and variables

Reply:
Ok sure